docs/workflow.md

```diff
@@ -0,0 +1,54 @@
+# Workflow 
+
+The Praqma workflow - often called The Phlow - is a combination of the Git extension and CI server for automation. In this the CI server is concourse with the git-phlow resource installed.
```

Comment:
it should be "a CI server" otherwise it implies that there is a git-phlow ci server.

Is our target audience for git phlow really using concourse or would it make a lot more sense in showing the Jenkins setup? I'm asking for a friend.